src/main/java/graphql/execution/async/AsyncExecutionStrategy.java

```diff
@@ -0,0 +1,131 @@
+package graphql.execution.async;
+
+import graphql.ExceptionWhileDataFetching;
+import graphql.ExecutionResult;
+import graphql.GraphQLError;
+import graphql.execution.ExecutionContext;
+import graphql.execution.ExecutionStrategy;
+import graphql.language.Field;
+import graphql.schema.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.function.Supplier;
+
+import static com.spotify.futures.CompletableFutures.successfulAsList;
```

Comment:
do you really want Spotify as a dependency.  I thought graphql-java was trying to be as lightweight as possible